contracts/gateway/OSTPrime.sol

```diff
@@ -121,7 +121,7 @@ contract OSTPrime is UtilityToken, OSTPrimeConfig {
      *
      * @return `true` if claim was successfully progressed
      */
-    function claim(
+    function unwrap(
```

Comment:
Function documentation and event name should also be updated with new terminology 'unwrap'.